services/app-web/src/pages/StateSubmission/RateDetails/RateDetails.tsx

```diff
@@ -216,6 +241,17 @@ export const RateDetails = ({
 
         draftSubmission.rateInfos = cleanedRateInfos
 
+        const addtlActuaryContacts: ActuaryContact[] = []
+        rateInfos.forEach((rateInfo) => {
+            rateInfo.addtlActuaryContacts.forEach((contact) => {
+                addtlActuaryContacts.push(contact)
+            })
+        })
+        draftSubmission.addtlActuaryContacts = addtlActuaryContacts
+
+        draftSubmission.addtlActuaryCommunicationPreference =
```

Comment:
Does the `addtlActuaryCommunicationPreference` need to be on each rate for flag off or does the single radio apply to all rates?